app/Lio/ServiceProviders/HashidsServiceProvider.php

```diff
@@ -5,15 +5,11 @@
 
 class HashidsServiceProvider extends ServiceProvider
 {
-    public function boot()
+    public function register() 
     {
-        $this->app->bind('Hashids\Hashids', function() {
+        $this->app->bind(['Hashids\Hashids' => 'hashids'], function() {
```

Comment:
AFAIK, `boot()` can used dependency injection, but not `register()`. And it seems to be called without any argument nor injection: https://github.com/laravel/framework/blob/5.0/src/Illuminate/Foundation/Application.php#L494